packages/public-api/src/apis/reddit/models/WikiPage.ts

```diff
@@ -97,9 +97,7 @@ export class WikiPage {
     this.#revisionId = data.revisionId;
     this.#revisionDate = new Date(data.revisionDate * 1000); // data.revisionDate is represented in seconds, so multiply by 1000 to get milliseconds
     this.#revisionReason = data.reason ?? '';
-
-    assertNonNull(data.revisionBy?.data, 'Wiki page author details are missing');
-    this.#revisionAuthor = new User(data.revisionBy.data, metadata);
+    this.#revisionAuthor = data.revisionBy?.data ? new User(data.revisionBy.data, metadata) : undefined;
```

Comment:
Are you able to run `yarn lint:fix` in the `packages/public-api` directory? Doing that should auto-format the code with line breaks like:

```
    this.#revisionAuthor = data.revisionBy?.data
      ? new User(data.revisionBy.data, metadata)
      : undefined;
```

Reply:
I couldn't get that to work, all it did was generate a `.filesystem.json` file. Instead I ran prettier separately and only accepted its formatting change to my code (but none of the formatting changes it wanted to make to unrelated code in WikiPage.ts).